lib/library.cpp

```diff
@@ -723,8 +723,14 @@ Library::Error Library::loadFunction(const tinyxml2::XMLElement * const node, co
                     else
                         wi.message += ", ";
                 }
-            } else
-                wi.message = functionnode->GetText();
+            } else {
+                const char * const message = functionnode->GetText();
+                if (!message) {
+                    printf("Error in library configuration. Warning for function '%s' is missing arguments 'reason' and 'alternatives' or some text.\n", name.c_str());
```

Comment:
this printf will not work super-good. GUI users will likely not see it. Feel free to extend the "Error" class.

Reply:
Good point